client/utils.c

```diff
@@ -649,3 +649,36 @@ TDNFGetCmdOpt(
 error:
     return dwError;
 }
+
+uint32_t
+TDNFYesOrNo(
+    PTDNF_CMD_ARGS pArgs,
+    const char *pszQuestion,
+    int *pAnswer
+    )
+{
+    uint32_t dwError = 0;
+    int nAnswer = 0;
+    int32_t opt = 0;
+
```

Comment:
Validate function parameters first and initialize *pAnswer = 0 before proceeding